Remaining time in ad break duration (#640)
bitmovin/bitmovin-player-ui: CHANGELOG.md

The format is based on [Keep a Changelog](http://keepachangelog.com/)
and this project adheres to [Semantic Versioning](http://semver.org/).

> ## [3.70.0] - 2024-08-21

Reviewer: As this is not yet released, the section should be named `Unreleased` and there should not be any date. See http://keepachangelog.com/

```suggestion
## [Unreleased]
```

### Added

> - Support for a new adMessagePlaceholder `adBreakRemainingTime` in [AdMessageLabel](https://cdn.bitmovin.com/player/ui/3/docs/classes/AdMessageLabel.html). It displays the remaining time in an ad break. [Documentation](https://cdn.bitmovin.com/player/ui/3/docs/functions/StringUtils.replaceAdMessagePlaceholders.html) on usage.

Reviewer: nit
```suggestion
- Support for a new placeholder `{adBreakRemainingTime}` in [AdMessageLabel](https://cdn.bitmovin.com/player/ui/3/docs/classes/AdMessageLabel.html) that displays the remaining time in an ad break. [Documentation](https://cdn.bitmovin.com/player/ui/3/docs/functions/StringUtils.replaceAdMessagePlaceholders.html) on usage.
```
